data/ovs/002-ovs-daemonset.yaml

```diff
@@ -44,6 +44,13 @@ spec:
         - name: ovs-cni-marker
           image: {{ .OvsMarkerImage }}
           imagePullPolicy: {{ .ImagePullPolicy }}
+          resources:
+            requests:
+              cpu: "100m"
```

Comment:
60m ?

Reply:
Did not work with 60m really, plus when I tried it manually, it was too slow.